tests/runner.py

```diff
@@ -32,7 +35,43 @@
 from osis_common.tests.functional.models.report import make_html_report
 
 
+class DebugTimeTextTestResult(unittest.TextTestResult):
```

Comment:
Etant donné que ton fichier s'appelle runner (et que la classe plus bas est un 'runner')
Met cette classe dans un fichier 'result' ? Ou renomme ce fichier ? La je fais mon chiant :D c'est à discuter

Reply:
Il s'agit d'une composante du runner, j'ai gardé la même structure que le built-in python 3.5 (package unittest/runner.py), les deux classes (**TextTestResult** ainsi que **TextTestRunner** se trouve dans le même fichier - https://github.com/python/cpython/blob/3.5/Lib/unittest/runner.py)